src/kernel-install/90-uki-copy.install

```diff
@@ -66,8 +66,8 @@ else
     UKI_FILE="$UKI_DIR/$ENTRY_TOKEN-$KERNEL_VERSION.efi"
 fi
 
-# If there is a UKI named uki.efi on the staging area use that, if not use what
-# was passed in as $KERNEL_IMAGE but insist it has a .efi extension
+# If there is a UKI named uki.efi on the staging area use that,
+# if not use what was passed in as $KERNEL_IMAGE
```

Comment:
Even with autodetection of UKIs, does it make sense to remove the convention?

Reply:
It looked like a sanity check to me.  With autodetection we know at this point that the given kernel image actually is a UKI (which implies it is a EFI PE executable), so the check IMHO doesn't make much sense any more.

We could continue enforce the `*.efi` naming convention.  Not sure what the point is, but if you prefer it that way I can drop the change.  I don't care much either way.

Reply:
https://uapi-group.org/specifications/specs/boot_loader_specification/#type-2-efi-unified-kernel-images is explicit: "the files must have the extension `.efi`". So let's keep the check for now.